### DISCUSSION TITLE
what about webhook option [May 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2021] [Discord Threads]
> 🙋‍♂️ **BoMBx**

**Comment by o1lab** [comment]
> 🙋‍♂️ **CaptainKranch**  
#### am I able to get information from an external api and put it on nocodb with the method get using webhooks? thanks for the reply!  
#  
> 🙋‍♂️ **BoMBx**  
#### Document here https://docs.nocodb.com/developer-resources/webhooks  
#  
> 🙋‍♂️ **BoMBx**  
#### what is your goal? If I can help you, I will.  
#  
> 🙋‍♂️ **CaptainKranch**  
#### I have around ~7k sim cards, this db is constantly updating. From those ~7k sim cards I need to activate them using a third party api. So my idea was, if the db was updated use the third party api to activate the new sim cards.  
#  
> 🙋‍♂️ **CaptainKranch**  
#### At the moment we are doing this on a gsheet with a custom script thourg extension --> app scripts but sometimes it is so slow that we are looking an alternative  
#  
> 🙋‍♂️ **CaptainKranch**  
#### can we do a call here on dc so you can help me? let me know when you are going to be available and I'll call you! Thanks a lot!  
#  
> 💁 **o1lab**  
#### I see there is a meeting you 've booked later - happy to be in touch and we will look into this  
#  
> 💁 **o1lab**  
#### apologies I missed the earlier schedule  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968336791184744519)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
